test/metabase/driver/firebolt_test.clj

```diff
@@ -59,15 +58,13 @@
   (testing "make sure the various types we use for running tests are actually mapped to the correct DB type"
      (are [db-type expected] (= expected
                                 (sql-jdbc.sync/database-type->base-type :firebolt db-type))
-                      :Int64      :type/BigInteger
-                      :UInt64     :type/BigInteger
-                      :Int        :type/Integer
-                      :Float      :type/Float
-                      :String     :type/Text
-                      :DateTime   :type/DateTime
-                      :Date       :type/Date
-                      :UUID       :type/UUID
-                      :Decimal    :type/Decimal)))
+                      :BIGINT :type/BigInteger
+                      :INTEGER :type/Integer
+                      :FLOAT :type/Float
+                      :STRING :type/Text
+                      :TIMESTAMP :type/DateTime
+                      :DATE :type/Date
+                      :DECIMAL :type/Decimal)))
```

Comment:
What about boolean in here?

Reply:
The current implementation of the JDBC driver does not support boolean (so we won't have that type)